### PR TITLE
Register GHG concentrations

### PR DESCRIPTION
**Pull request by znichollscr**
@durack1 I wasn't sure exactly what to fill in where here. Let's discuss, fix, then I can write docs and include in here to help people.

### CODE REVIEW

File: input4MIPs_source_id.json [hunk 1]
```diff
@@ -2869,6 +2869,22 @@
             "version":"20181127",
             "xlink":"http://cera-www.dkrz.de/WDCC/meta/CMIP6/input4MIPs.CMIP6.AerChemMIP.UoM.UoM-AIM-ssp370-lowNTCF-1-2-1.atmos.yr.mole_fraction_of_sf6_in_air.gr1-GMNHSH.20181127.json|Citation|citation"
         },
+        "CR-DECK-0-1-0":{
+            "contact":"zebedee.nicholls@climate-resource.com",
```
Comment (znichollscr) [inline input4MIPs_source_id.json:2873]: @durack1 which fields are compulsory and which are optional?

Is it worth adding an automated check (GitHub CI) that makes sure that what users add here is valid/correct?

File: README.md [hunk 2]
```diff
@@ -10,6 +10,16 @@ To see information regarding forcing dataset development for the Coupled Model I
 
 To register your institution or source/dataset, please submit an issue/ticket following the instructions on the [input4MIPs_CVs issue page](https://github.com/PCMDI/input4MIPs_CVs/issues/new). Follow the same procedure to request a change in any other CV.
 
+### Registering a new input4MIPs forcing dataset
+
+[ZN to write, rough steps, current guess below]
```
Comment (znichollscr) [inline README.md:15]: @durack1 not sure if you have other thoughts on whether this information should live here or somewhere else, but obviously we need to capture the 'how do I get my data onto ESGF' process somewhere

**Comment by durack1** [comment]
@znichollscr might be best to move this chatter into an issue, as we have much information to coalesce across data products into a general purpose template - just FYI/and for my recollection a first pass attempt a synthesizing this across PCMDI-AMIP-1-1-9 (first CMIP6Plus dataset), CR-CMIP-0-1-0 (new) and CEDS-2017-05-18-CO2-em (CMIP6 dataset) is [here](https://docs.google.com/spreadsheets/d/1vMlMVAnP25GpHcS0B8cyvFvUoTZxlUvxBmFB-PbLDG0/edit#gid=108565686). The information in the current [input4MIPs_source_id.json](https://github.com/PCMDI/input4MIPs_CVs/blob/main/input4MIPs_source_id.json), was just a subset pulled from the ESGF index, as below:

https://github.com/PCMDI/input4MIPs_CVs/blob/903ec2bd1c5e925a3fa5e610a4414f131a95e2bc/input4MIPs_source_id.json#L2131-L2149

In a near-final version, I would prefer augmented info, similar to what I have for the latest `PCMDI-AMIP-1-1-9` dataset, which was generated from:

```python
        "PCMDI-AMIP-1-1-9":{
            "calendar":"gregorian",
            "comment":"Based on Hurrell SST/sea ice consistency criteria applied to merged HadISST (1870-01 to 1981-10) & NCEP-0I2 (1981-11 to 2022-12)",
            "contact":"PCMDI (pcmdi-cmip@llnl.gov)",
            "dataset_category":"SSTsAndSeaIce",
            "further_info_url":"https://pcmdi.llnl.gov/mips/amip",
            "grid":"1x1 degree longitude x latitude",
            "grid_label":"gn",
            "institution":"Program for Climate Model Diagnosis and Intercomparison, Lawrence Livermore National Laboratory, Livermore, CA 94550, USA",
            "institution_id":"PCMDI",
            "license":"AMIP boundary condition data produced by PCMDI is licensed under a Creative Commons Attribution 4.0 International License (CC BY 4.0; https://creativecommons.org/licenses/by/4.0). Consult https://pcmdi.llnl.gov/CMIP6/TermsOfUse for terms of use governing input4MIPs output, including citation requirements and proper acknowledgment. Further information about this data, including some limitations, can be found via the further_info_url (recorded as a global attribute in this file). The data producers and data providers make no warranty, either express or implied, including, but not limited to, warranties of merchantability and fitness for a particular purpose. All liabilities arising from the supply of the information (including any liability arising in negligence) are excluded to the fullest extent permitted by law",
            "mip_era":"CMIP6Plus",
            "nominal_resolution":"1x1 degree",
            "product":"observations",
            "references":"Taylor, K.E., D. Williamson and F. Zwiers, 2000: The sea surface temperature and sea ice concentration boundary conditions for AMIP II simulations. PCMDI Report 60, Program for Climate Model Diagnosis and Intercomparison, Lawrence Livermore National Laboratory, 25 pp. Available online: https://pcmdi.llnl.gov/report/pdf/60.pdf",
            "region":[
                "global_ocean"
            ],
            "release_year":"2023",
            "source":"PCMDI-AMIP 1.1.9: Merged SST based on UK MetOffice HadISST and NCEP OI2",
            "source_description":"Sea surface temperature and sea-ice datasets produced by PCMDI (LLNL) for the AMIP (DECK) experiment of CMIP6Plus",
            "source_id":"PCMDI-AMIP-1-1-9",
            "source_type":"satellite_blended",
            "source_variables":[
                "areacello",
                "sftof",
                "siconc",
                "siconcbcs",
                "tos",
                "tosbcs"
            ],
            "source_version":"1.1.9",
            "target_mip":"CMIP",
            "title":"PCMDI-AMIP 1.1.9 dataset prepared for input4MIPs"
        },
```

Which was recorded in the previous iteration repo [PCMDI/input4MIPs-cmor-tables/input4MIPs_source_id.json](https://github.com/PCMDI/input4MIPs-cmor-tables/blob/fa10c47fef22b0ffab05615d93cbe6de328ccb2c/input4MIPs_source_id.json#L801-L835)

The above (PCMDI-AMIP-1-1-9 example from the input4MIPs-cmor-tables repo) may also need an entry that details changes between the previous 1.1.8 and the current 1.1.9 data, along with the fact that this is the "endorsed" AMIP SST and sea ice dataset (rather than some secondary prototype/testing dataset that can be optionally used), along with any other information/identifiers that data providers want to include about their data. In the case that certain fields aren't used by some data providers, these can remain blank.

**Comment by znichollscr** [comment]
> In a near-final version, I would prefer augmented info, similar to what I have for the latest `PCMDI-AMIP-1-1-9` dataset, which was generated from

Ok I think that makes more sense. To check I understand, this source ID file would define not only high-level info about a dataset, but also details e.g. the grid and calendar?

Assuming I've understood correctly, we would then need a different source ID for each product that is on a different grid/calendar. For example, we provide data as a global-mean (no grid basically) and on a 15x360 degree grid (i.e. 15 degree latitudinal bands). With the proposal above, we would need to register two source IDs to differentiate between those two. Is that on purpose or is it better to drop some keys (that are data file, rather than data set, specific) out?

**Comment by znichollscr** [comment]
Made #13 to continue the discussion

**Comment by durack1** [comment]
Closing in favour of disussions in #13

**Pull request closed without merging**